package.json

```diff
@@ -1,12 +1,14 @@
 {
   "name": "fx-private-relay-add-on",
-  "version": "2.1.1",
+  "version": "2.2.0",
   "description": "Firefox Relay",
   "main": "index.js",
   "dependencies": {
     "web-ext": "^5.2.0"
   },
-  "devDependencies": {},
+  "devDependencies": {
```

Comment:
Note that I added this here, even though I also hard-coded the library into the repo. I wanted to add it here for dependency tracking.

Reply:
Hmm — I'm going to remove for now from the dependencies list, but file an issue to move it into a build step.

Reply:
Removed in 302fa9917ade376339068731a4812e7e542d0012